### PR TITLE
Issue deprecation warning for Python<3.9

### PR DESCRIPTION
**Pull request by fwitte**
As discussed in #101

### CODE REVIEW

**Comment by fwitte** [comment]
Is this a correct way to issue warnings? Or does it only work if someone directly imports `oemof.tabular` and not `from oemof.tabular`?

File: src/oemof/tabular/__init__.py [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+
+
+import sys
+import warnings
+
+major, minor, micro = sys.version_info[:3]
+if minor < 9:
+    warnings.warn(
+        f"You are using python version {major}.{minor}.{micro}. oemof.tabular "
+        "will stop supporting python < 3.9 in the future."
```
Comment (nailend) [inline src/oemof/tabular/__init__.py:12]: I would also use a `DepricationWarning` but somehow, this is no displayed in the console without setting `stacklevel=2`?!
```
    msg = f"You are using python version 3.8./n"
          "oemof.tabular will stop supporting python 3.8 in the future."
    warnings.warn(msg, DeprecationWarning, stacklevel=2)
```

**Review by nailend — APPROVED** [review]
I made some comments, but your version is totally sufficient. Thanks for the proposal.

**Comment by fwitte** [comment]
Thanks for the suggestions, I pushed the changes. I do not consider myself an active user/developer of oemof.tabular. Therefore I would like others to merge or not to merge this. So go ahead as you like, same for #101

**Pull request merged**